src/ui/shared/forms/SuccessState/SuccessStateLoader.tsx

```diff
@@ -0,0 +1,317 @@
+      <UIText kind="headline/hero">.</UIText>
+      <UIText kind="headline/hero">.</UIText>
+      <UIText kind="headline/hero">.</UIText>
+    </HStack>
+  );
+}
+
+function BackgroundDecoration({ color }: { color: string | null }) {
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        top: '-30vh',
+        left: 0,
+        right: 0,
+        height: '60vh',
+        transitionDuration: '1s',
+        willChange: 'opacity',
+        opacity: color ? 0.7 : 0,
+        background: `radial-gradient(circle at top center, ${color}, transparent 65%)`,
+      }}
+    />
+  );
+}
+
+export function SuccessStateLoader({
+  startItem,
+  endItem,
+  status,
+  pendingTitle = 'Pending',
+  confirmedTitle = 'Done',
+  failedTitle = 'Failed',
+  dropppedTitle = 'Dropped',
+  explorerUrl,
+  error,
+  confirmedContent,
+  onDone,
+}: {
+  startItem: React.ReactNode;
+  endItem: React.ReactNode;
+  status: ClientTransactionStatus;
+  pendingTitle?: string;
+  confirmedTitle?: string;
+  failedTitle?: string;
+  dropppedTitle?: string;
+  explorerUrl?: string;
+  error?: string;
+  confirmedContent?: React.ReactNode;
+  onDone?: () => void;
+}) {
+  const showLongWaitNotice = useRenderDelay(5000);
+
+  const backgroundColor =
+    status === 'confirmed'
+      ? 'var(--positive-500)'
+      : status === 'failed' || status === 'dropped'
+      ? 'var(--negative-500)'
+      : null;
+
+  return (
+    <PageColumn
+      style={{
+        width: 'clamp(320px, 100vw, 450px)',
+        marginInline: 'auto',
+        position: 'relative',
+      }}
+    >
+      <BackgroundDecoration color={backgroundColor} />
+      <VStack
+        gap={0}
+        style={{
+          gridTemplateRows: '1fr auto auto',
+          alignItems: 'center',
+          flexGrow: 1,
+          position: 'relative',
+        }}
+      >
+        <VStack
+          gap={32}
+          style={{ justifyItems: 'center', justifySelf: 'center' }}
+        >
+          <AnimatedIcons
+            startItem={startItem}
+            endItem={endItem}
+            status={status}
+          />
+          <VStack gap={16} style={{ justifyItems: 'center' }}>
+            <VStack gap={8} style={{ justifyItems: 'center' }}>
+              <UIText kind="headline/hero" style={{ position: 'relative' }}>
+                {NBSP}
+                <HStack
+                  gap={4}
+                  className={cn(
+                    styles.title,
+                    status !== 'pending' && styles.gone
+                  )}
+                >
+                  <UIText kind="headline/hero">{pendingTitle}</UIText>
+                  <AnimatedDots />
+                </HStack>
+                <UIText
+                  kind="headline/hero"
+                  className={cn(
+                    styles.title,
+                    status !== 'confirmed' && styles.hidden
+                  )}
+                >
+                  {confirmedTitle}
+                </UIText>
```

Comment:
I know it's minor but I don't like that text selection is messed up in this area because of overlapping text elements with zero opacity
Can we fix this?

Reply:
Good point!